2022 - LA - Sons of Horus.cat

```diff
@@ -4643,7 +4643,7 @@
       </constraints>
       <rules>
         <rule id="cc22-87b0-6a3b-bbb6" name="Born Survivor" hidden="false">
-          <description>The first time in any game when a model with this special rule is reduced to 0 Wounds for any reason or otherwise removed from play as a casualty, the controlling player must immediately make a Leadership test for that model. If the test is failed then the model is removed as a casualty as normal, but if the test is passed then the Warlord is not removed as a casualty, remains in play and regains 1D3 Wounds.</description>
+          <description>The first time in any game when a model with this special rule is reduced to 0 Wounds for any reason or otherwise removed from play as a casualty, the controlling player must immediately make a Leadership test for that model. If the test is failed then the model is removed as a casualty as normal, but if the test is passed then the Warlord is not removed as a casualty, remains in play and regains 1D3 Wounds.</description>
```

Comment:
```suggestion
          <description>The first time in any game when a model with this special rule is reduced to 0 Wounds for any reason or otherwise removed from play as a casualty, the controlling player must immediately make a Leadership test for that model. If the test is failed then the model is removed as a casualty as normal, but if the test is passed then the Warlord is not removed as a casualty, remains in play and regains D3 Wounds.</description>
```
D3 is the official templating not 1D3